### DISCUSSION TITLE
Oh dear, Another crash. This time this is not a RDP.

### DISCUSSION THREAD

**Discussion by LuisYeah1234-hub** [Maplespe/DWMBlurGlass#166] [Q&A]
![image](https://github.com/Maplespe/DWMBlurGlass/assets/64372171/1dadb868-1179-426c-84c5-aa7a29779c46)
A crash caused by dwmblurglassext dll
Dont know if this is a duplicate of another issue

**Comment by Maplespe** [comment]
You will need to provide me with more information, for example:
Are you using the custom blur method, and what did you do before the crash?
Did it crash after booting up or during use.
What other software did you use it with?
We need more information.

**Comment by LuisYeah1234-hub** [comment]
1. No, And was using programs completely. Chrome
2. After use
3. Don't remember, but just normal programs like cmd, chrome etc

**Comment by ALTaleX531** [comment]
OldBlur is very easy to cause crashes and we may not maintain this blur method in the future.

**Comment by LuisYeah1234-hub** [comment]
> OldBlur is very easy to cause crashes and we may not maintain this blur method in the future.

OldBlur causes crashes??
and no im using CustomBlur

**Comment by ALTaleX531** [comment]
Oh sorry I thought your first reply meant you don't use CustomBlur, never mind.

**Comment by LuisYeah1234-hub** [comment]
> Oh sorry I thought your first reply meant you don't use CustomBlur, never mind.

thats ok
But this happened suddenly so it may be hard to reproduce.

**Comment by LuisYeah1234-hub** [comment]
It happened again this time it was reproduced during startup
